scss/index.js

```diff
@@ -11,14 +11,17 @@ module.exports = {
     'scss/at-function-pattern': '^[a-z][a-z-]*$',
     'scss/at-function-parentheses-space-before': 'never',
     'scss/at-import-no-partial-leading-underscore': true,
+    'scss/at-import-partial-extension': 'never',
     'scss/at-mixin-argumentless-call-parentheses': 'always',
     'scss/at-mixin-named-arguments': null,
     'scss/at-mixin-parentheses-space-before': 'never',
     'scss/at-rule-no-unknown': true,
+    'scss/dimension-no-non-numeric-values': true,
     'scss/dollar-variable-colon-space-after': 'at-least-one-space',
     'scss/dollar-variable-colon-space-before': 'never',
     'scss/dollar-variable-no-missing-interpolation': true,
     'scss/dollar-variable-pattern': '^[a-z][a-z-\\d]*$',
+    'scss/map-keys-quotes': null, // This doesn't work well yet
```

Comment:
What does _This doesn't work well yet_ mean?

Reply:
IIRC we had some false positives last time I tried it hence why I disabled it.